packages/blazor-workspace/NimbleBlazor/README.md

```diff
@@ -113,6 +87,30 @@ See the [Blazor prerendering docs](https://learn.microsoft.com/en-us/aspnet/core
 
 To use Nimble's theme-aware design tokens in a Blazor app, you should have a `<NimbleThemeProvider>` element as an ancestor to all of the Nimble components you use. The app's default layout (`MainLayout.razor` in the examples) is a good place to put the theme provider (as the root content of the page).
 
+#### Best practices
+
+Custom Blazor components should provide their own scoped CSS file (in addition to a separate .cs file for the template-independent logic). Providing a separate CSS file is necessary to access other Blazor styling mechanisms that are helpful to use.
+
+#### Styling Blazor components
+
+Often you will need to provide CSS for the Nimble Blazor components (et. al) to control things like layout behaviors within a parent container. To accomplish this, in the scoped CSS file for the component containing the Nimble Blazor component (e.g. NimbleTextField), you must use the `::deep` pseudo-selector to target that Nimble component.
```

Comment:
What is "et. al" meant to convey here? That this is necessary for more types of components than just Nimble Blazor? Could we be more explicit? For example "Nimble and Spright Blazor components" if this is a FAST-specific thing or "child components" if this is a Blazor-specific thing.

Reply:
Removed that wording and am now using the more general "Razor term", as this guidance is meant to cover _any_ Razor component.